caffe2/operators/onnxifi_op.cc

```diff
@@ -0,0 +1,192 @@
+#include "caffe2/operators/onnxifi_op.h"
+
+namespace caffe2 {
+
+namespace {
+
+void CPUTensorToTensorProto(
```

Comment:
You don't need this function: `TensorProto` is only used for `initializer`, and we no longer use it.